src/App.js

```diff
@@ -33,17 +32,38 @@ const App = () => (
       <p className="Person__age">I am 37</p>
       <p className="Person__partner">My wife&apos;s name is Natasha</p>
     </section>
+    <Person
+      name={misha.name}
+      age={misha.age}
+      sex={misha.sex}
+      isMarried={misha.isMarried}
+      partnerName={misha.partnerName}
+    />
 
     <section className="Person">
       <h2 className="Person__name">My name is Olya</h2>
       <p className="Person__partner">My husband&apos;s name is Maksym</p>
     </section>
+    <Person
+      name={olya.name}
+      age={olya.age}
+      sex={olya.sex}
+      isMarried={olya.isMarried}
+      partnerName={olya.partnerName}
+    />
```

Comment:
```suggestion
    <Person
      name={olya.name}
      age={olya.age}
      sex={olya.sex}
      isMarried={olya.isMarried}
      partnerName={olya.partnerName}
    />
```
why do you need this dublicate